src/lib/phdi-building-blocks/phdi_building_blocks/standardize.py

```diff
@@ -1,93 +1,172 @@
 import phonenumbers
 import pycountry
 from typing import Callable, Literal, List
-
 from phdi_building_blocks.utils import find_patient_resources
+import copy
 
 
-def standardize_name(raw: str) -> str:
-    """trim spaces and force uppercase
-
-    >>> standardize_name(" JohN Doe ")
-    'JOHN DOE'
+def non_numeric_caps_standardization(raw_name: str) -> str:
     """
-
-    raw = [x for x in raw if not x.isnumeric()]
-    raw = "".join(raw)
-    raw = raw.upper()
-    raw = raw.strip()
-    return raw
+    Perform standardization on a provided name string. Name
```

Comment:
We've changed the name of the function, but the docstring indicates it's still just for names. If we're trying to generalize this to handle any type of text, then I recommend we do something like the following:

```python
def standardize_text(
    text: str, 
    remove_numbers=False, 
    case=Literal["upper", "lower", "title", "original"]="original",
    <other parameters>
) -> str:
    <code>
```
And then we remove all references to name and anything else, and we move it to `utils.py`. Hell, there are probably already functions like this that we could just import.